### PR TITLE
improvement: OSIS-51-use-scality-package-and-remove-stub-code

### PR DESCRIPTION
**Pull request by anurag4DSB**
This PR 
- migrates the AppEnv and packages/ classes required for user authentication
- removed Vmware stub code.

### CODE REVIEW

**Comment by anurag4DSB** [comment]
build started again as force pushed a fixup

File: jacoco.gradle [hunk 1]
```diff
@@ -75,7 +75,7 @@ jacocoTestCoverageVerification {
     violationRules {
         rule {
             limit {
-                minimum = 0.8 // verifying mandatory code coverage of 80%
+                minimum = 0.75 // verifying mandatory code coverage of 80%
```
Comment (XinLiScality) [inline jacoco.gradle:78]:
```suggestion
                minimum = 0.75 // verifying mandatory code coverage of 75%
```
😄

File: osis-core/src/main/java/com/scality/osis/annotation/NotImplement.java [hunk 2]
```diff
@@ -1,9 +1,10 @@
 /**
  *Copyright 2020 VMware, Inc.
```
Comment (XinLiScality) [inline osis-core/src/main/java/com/scality/osis/annotation/NotImplement.java:2]: can we remove this line if we don't use vmware packages

**Review by XinLiScality — APPROVED** [review]

File: osis-core/src/main/java/com/scality/osis/service/ScalityOsisService.java [hunk 3]
```diff
@@ -6,10 +6,67 @@
 
 package com.scality.osis.service;
 
-import com.vmware.osis.model.OsisS3Credential;
-import com.vmware.osis.service.OsisService;
+import com.scality.osis.model.*;
+import com.scality.osis.model.ScalityOsisCaps;
+import com.scality.osis.model.OsisS3Credential;
```
Comment (williamlardier) [inline osis-core/src/main/java/com/scality/osis/service/ScalityOsisService.java:11]: Are the two last lines needed if you import all the namespace with the star?

**Review by williamlardier — APPROVED** [review]
LGTM, just a small comment (maybe I'm wrong, I haven't used java for a while :smile: )

**Review by WillToozsScality — APPROVED** [review]

File: osis-core/src/main/java/com/scality/osis/annotation/NotImplement.java (hunk 2 above)
Comment (anurag4DSB) [reply osis-core/src/main/java/com/scality/osis/annotation/NotImplement.java:2]: Its still VMware code, so we should keep it.
Most of it is copied over and modified, so we need to keep both licencnes.

File: osis-core/src/main/java/com/scality/osis/service/ScalityOsisService.java (hunk 3 above)
Comment (anurag4DSB) [reply osis-core/src/main/java/com/scality/osis/service/ScalityOsisService.java:11]: you are right, it was very intentional to not change that.
We will refactor the whole code base again soon. For now, so close to the release, I didn't want to add more complexities in the review.

File: jacoco.gradle (hunk 1 above)
Comment (anurag4DSB) [reply jacoco.gradle]: fixed and force pushed.
thanks :)

**Pull request merged**